web/packages/teleport/src/Discover/Database/EnrollRdsDatabase/EnrollRdsDatabase.tsx

```diff
@@ -413,15 +439,45 @@ function getRdsEngineIdentifier(engine: DatabaseEngine): RdsEngineIdentifier {
   }
 }
 
+const discoveryGroupToolTip = `Discovery group name is used to group discovered resources into different sets. \
+This parameter is used to prevent Discovery Agents watching different sets of cloud resources from \
+colliding against each other and deleting resources created by another services.`;
+
```

Comment:
This reads a little weird. But tbh, i haven't used this in production so idk. what are the different "sets", is that the technical term or is there a better word we can use here. Also, "parameter" doesn't really seem UX friendly. Maybe "value" or something but idk, I won't die on that hill. 

I don't recommend my suggestion 100%, but here are my thoughts.

"Discovery Group Name (i know its not all capital but maybe it'll help differentiate it as the `term`) is used to name a group of discovered resources. This group name is used to prevent deleting resources created by other services being watched by different Discovery Agents"

could fiddle with it idk

Reply:
i copied this directly from the docs, so i'm hesitant to change it since it was reviewed by the doc folks: https://docs-2dew2l61w-goteleport.vercel.app/docs/database-access/guides/aws-discovery/#step-24-configure-the-discovery-service

Reply:
Sounds good to me